src/components/Navigation/Navigation.js

```diff
@@ -57,6 +57,7 @@ const Navigation = (props) => {
     const {pathname} = useLocation();
     const dispatch = useDispatch();
     const {token, isAdmin} = useSelector(({auth}) => auth);
+    console.log(token);
```

Comment:
Delete this. If you need to see the token, go to your redux tools and click on "state" 
<img width="1485" alt="Screen Shot 2020-01-03 at 9 31 10 PM" src="https://user-images.githubusercontent.com/12959959/71760331-d8660180-2e70-11ea-9498-8cc012ea5da3.png">